### PR TITLE
add get http query uri

### PR DESCRIPTION
**Pull request by williballenthin**
ref #3

please LINT_THOROUGH

### CODE REVIEW

File: communication/http/get-http-query-uri.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      - william.ballenthin@mandiant.com
+    scope: basic block
+    mbc:
+      - Communication::HTTP Communication [C0002]
+    examples:
+      - Practical Malware Analysis Lab 03-02.dll_:0x10004452
+  features:
+    - and:
+      - api: wininet.HttpQueryInfo
+      - number: 0x13 = HTTP_QUERY_URI
```
Comment (mr-tz) [inline communication/http/get-http-query-uri.yml:15]: Should this be 0xD (decimal 13) via https://learn.microsoft.com/en-us/windows/win32/wininet/query-info-flags?

File: communication/http/get-http-query-uri.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: get HTTP query URI
```
Comment (mr-tz) [inline communication/http/get-http-query-uri.yml:3]: I'm not sure about "query"... How about request or header?

File: communication/http/get-http-query-uri.yml (hunk 1 above)
Comment (williballenthin) [reply communication/http/get-http-query-uri.yml:15]: yes, definitely.

1) 🤦🏼
2) should have run with `lint --thorough` locally, but forgot
3) why didn't CI catch this with `--thorough`?

File: communication/http/get-http-query-uri.yml (hunk 2 above)
Comment (williballenthin) [reply communication/http/get-http-query-uri.yml:3]: tbh, i misunderstood what this `HttpQueryInfo` option fetched. i thought it was the query parameters portion of the URL, but i see now its the whole URL. so i'd propose:

`get HTTP request URL`

File: communication/http/get-http-query-uri.yml (hunk 1 above)
Comment (williballenthin) [reply communication/http/get-http-query-uri.yml:15]:
```suggestion
      - number: 0xD = HTTP_QUERY_URI
```

Comment (williballenthin) [reply communication/http/get-http-query-uri.yml:15]: ah i need `LINT_THOROUGH` in the PR text.

**Comment by mr-tz** [comment]
Hm, the linter should have run thoroughly before. Maybe there's a 13 in the block as well?

**Review by mr-tz — APPROVED** [review]
😃 cool

**Pull request merged**